src/config.h.cmake

```diff
@@ -75,9 +78,15 @@
 /* Define to 1 if you have the <stdint.h> header file. */
 #cmakedefine HAVE_STDINT_H
 
+/* Define to 1 if you have the <stdlib.h> header file. */
+#cmakedefine HAVE_STDLIB_H
```

Comment:
not sure why these are popping back up ... we cleaned them out.  rebase onto the latest tree and re-run the bootstrap script to see if these get cleaned up.